src/styles/resets.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
it is better not to add margin with * as it may cause further problems

Reply:
Thank! I will not allow this in future projects.

Reply:
Fix it here as well.

Reply:
done